models/correlation_detector.cpp

```diff
@@ -54,13 +56,15 @@ nest::correlation_detector::Parameters_::Parameters_( const Parameters_& p )
   , Tstart_( p.Tstart_ )
   , Tstop_( p.Tstop_ )
 {
-  // Check for proper properties is not done here but in the
-  // correlation_detector() copy c'tor. The check cannot be
-  // placed here, since this c'tor is also used to copy to
-  // temporaries in correlation_detector::set_status().
-  // If we checked for errors here, we could never change values
-  // that have become invalid after a resolution change.
-  delta_tau_.calibrate();
+  if ( not delta_tau_.is_step() )
+  {
+    delta_tau_ = 5 * Time::get_resolution();
```

Comment:
Should this emit a warning?

Reply:
I don't think so.

There is already an info message when the resolution is actually changed. The adaptation here only updates the default (to the same value that was already infoed to the user) when models are reinstated from pristine prototypes (that still might have old values) upon changing the number of threads and if the very first model created happens to be one that can adapt default values automatically.

I consider all of these cases technical details the user does not need to know about and which will probably change again in the next installment of the module refactoring series of PRs.

Reply:
OK, thanks for the explanation.